charts/generic-data-analytics-extractor/values.yaml

```diff
@@ -33,6 +33,11 @@ dataPlatformApiAuthSecretKey: auth-key
 # a registered data product
 dataProductName: ""
 
+# Optional chart name override. The default is the release name (truncated to 27) followed by `-data-analytics-extractor`
+nameOverride: ""
+
+fullnameOverride: ""
```

Comment:
are you using this?

Reply:
Thats a fair comment. No, not strictly (or at all 😁 ), I'm not using it.
But I did notice it is being used already in `_helpers.tpl` in the `app.fullname` macro:
```
{{/*
Create a default fully qualified app name.
We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
If release name contains chart name it will be used as a full name.
*/}}
{{- define "app.fullname" -}}
{{- if .Values.fullnameOverride -}}
{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
{{- else -}}
{{- $name := default .Chart.Name .Values.nameOverride -}}
{{- if contains $name .Release.Name -}}
{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
{{- else -}}
{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
{{- end -}}
{{- end -}}
{{- end -}}
```
I noticed it because my IDE highlighted it as not being defined in `values.yaml`